package.json

```diff
@@ -30,7 +30,8 @@
   "contributors": [
```

Comment:
remove this block completely plz, or put all contributors, cause github counted at least 10

Reply:
nevermind we're abandoning kardia, closed that PR

Reply:
we're not using kardia anymore fyi

Reply:
by we, you mean who?

Reply:
dev teams in Lisbon

Reply:
https://github.com/search?q=org%3Apipedrive+kardia&type=Code

Reply:
what do you use instead?

Reply:
for metrics prometheus client, for health check an endpoint in carrier

Reply:
Ahhaa, will try the prometheus